docs/Developers/Development_Setup.rst

```diff
@@ -63,12 +63,13 @@ source in the future.
 Building Documentation
 ----------------------
 
-You will need to install the following packages to build the documentation:
+You will need to install the following packages after entering ``docs``
+directory to build the documentation:
 
 ::
 
-    (venv)$ pip3 install sphinx sphinx_rtd_theme
+    (venv)$ pip3 install -r docs-requirements.txt
 
-Once you have done so, you can build the documentation by entering the docs
-directory and running ``make``. The documentation on the coala website is in
-the ``coala`` (not ``coala-bears``) repository.
+Once you have done so, you can build the documentation by  running ``make``.
```

Comment:
In "The documentation on the coala website.. " How about hyperlink `coala website`  to  go on `https://docs.coala.io/en/latest/` ?